src/sidebar/components/SidebarTabbedPanel.tsx

```diff
@@ -15,6 +15,11 @@ import { applyTheme } from '../helpers/theme';
 import { withServices } from '../service-context';
```

Comment:
I suggest the name `SidebarTabs` for this component. In our own UI terminology, a "panel" is a thing that shows up at the top of the sidebar, whereas this occupies the whole content area. In WAI-ARIA terminology, the tab list + panel together is called the "tabs" pattern, the "tab list" is the list of tabs at the top and the "tab panel" is the content.

Reply:
The reasoning makes sense. I was not fully convinced with `SidebarTabbedPanel`.